packages/imitation_game/imitation_tests/smiley/android/run_android.sh

```diff
@@ -0,0 +1,14 @@
+set -e
+cd $( dirname "${BASH_SOURCE[0]}" )
+source ../../../helper/run_android_helper.sh
+
+readonly BUNDLE="dev.flutter.imitation_game.smiley"
+readonly TEST_NAME="smiley"
+
+# TODO(gaaclarke): Get assembleRelease working.  I don't know if the performance
+# is any different but assembleRelease requires signing the apk.
```

Comment:
No it doesn't https://github.com/flutter/flutter/blob/eef4220a878fd5dfe1551f76a78168347d6aa9eb/examples/hello_world/android/app/build.gradle#L47

Reply:
If I use assembleRelase I get this error:
```
aaclarke-macbookpro2:android aaclarke$ adb install ./app/build/outputs/apk/release/app-release-unsigned.apk
Performing Streamed Install
adb: failed to install ./app/build/outputs/apk/release/app-release-unsigned.apk: Failure [INSTALL_PARSE_FAILED_NO_CERTIFICATES: Package /data/app/vmdl1987488777.tmp/base.apk has no certificates at entry AndroidManifest.xml]
```

Googling that error leads to a myriad of suggestions.

Reply:
Got help via chat, thanks fixed it, works for assembleRelease now.